thirdeye-ui/src/app/utils/routes/routes.util.ts

```diff
@@ -571,3 +578,23 @@ export const generateDateRangeMonthsFromNow = (
 
     return [xMonthsAgo.toMillis(), roundedNow.toMillis()];
 };
+
+/**
+ * Helper function to quickly generate a date range for any number of days from
+ * now. The month ago will start at the beginning of the days.
```

Comment:
`The days ago will start at the beginning of the day.`

Reply:
Oops. My bad 😅.
Fixed